addons/product/data/product_data.xml

```diff
@@ -20,6 +20,10 @@
             <field name="name">Product Price</field>
             <field name="digits">2</field>
         </record>
+        <record forcecreate="True" id="decimal_variant_attribute_price" model="decimal.precision">
+            <field name="name">Value Price Extra</field>
+            <field name="digits">2</field>
+        </record>
```

Comment:
`price_extra` field uses the 'Product Price' decimal precision, why do you add this record ?

Reply:
Indeed, my bad.
(Forgot to remove it after reproduction and left it there + msg :facepalm: )